doc/lua_api.md

```diff
@@ -8843,6 +8843,16 @@ Player properties need to be saved manually.
     show_on_minimap = false,
     -- Defaults to true for players, false for other entities.
     -- If set to true the entity will show as a marker on the minimap.
+
+    engine_mask = {
+    -- This can be used for enable/disable some part of engine.
+      drowning = true,
+      -- Drowning (Only avaliable for players)
+      breathing = true,
+      -- Breathing (Only avaliable for players)
+      node_hurt = true,
+      -- Node damage (Only avaliable for players)
+    },
```

Comment:
just spelling and stuff

```suggestion
    engine_mask = {
    -- This can be used to enable/disable some parts of the engine.
      drowning = true,
      -- Drowning (only available for players)
      breathing = true,
      -- Breathing (only available for players)
      node_hurt = true,
      -- Node damage (only available for players)
    },
```